src/core/Events.js

```diff
@@ -96,30 +96,22 @@ L.Evented = L.Class.extend({
 		var typeListeners = this._events[type];
 		if (!typeListeners) {
 			typeListeners = {
-				listeners: {},
+				listeners: [],
 				count: 0

```

Comment:
I think this count variable can be removed and replaced by `listeners.length`. 

Actually I think the whole `typeListeners` object can be replaced by the `listeners` array now.